eth/hardhat.config.ts

```diff
@@ -36,10 +37,24 @@ require('dotenv').config();
 
 const { DEPLOYER_MNEMONIC, ADMIN_PUBLIC_ADDRESS } = process.env;
 
+const contracts = workspace('@darkforest_eth/contracts');
+if (!contracts) {
+  throw new Error('Unable to locate `@darkforest_eth/contracts` workspace');
+}
```

Comment:
I was on the fence about these but I thought it better to have explicit errors if the `package-lock` file is messed up instead of using `as string` to lie to typescript

Reply:
sounds good. I guess the only annoying thing here is if we are using a different npm org like `@dfdao` or `@zkgame`. Is there a better way to handle that case without hardcoding the new package?

Reply:
Workspaces use the `name` field in the `package.json` files within each to identify the package. The reason for this is to treat them as though they were installed via the npm registry (even though they are symlinked during development). If you change the `scope` (e.g. `@dfdao`) then you need to update all references to those throughout the project. There's no way around that.